### PR TITLE
Add support for nested grid reducers

### PR DESCRIPTION
**Pull request by drownbes**
reducerKeys now could be string that means
that root grid reducers is on the state[reducersKeys]
position in app state

### CODE REVIEW

**Comment by bencripps** [comment]
Thanks for the PR!

Could you give a brief description of how this changes the API for grid? Could you provide examples of how you would instantiate grid -- and does it still work both ways?

This version still works, correct? 

```js
import { combineReducers } from 'redux';
import { Reducers as gridReducers } from 'react-redux-grid';

import myAppReducer from './customReducers/app';
import myDataReducer from './customReducers/data';

export const rootReducer = combineReducers({
    myAppReducer,
    myDataReducer,
    ...gridReducers
});
```

How do you envision people using this new method?

File: src/util/stateGetter.js [hunk 1]
```diff
@@ -10,15 +10,22 @@
 
 export const stateGetter = (state, props, key, entry) => {
 
-    if (props
-        && props.reducerKeys
-        && Object.keys(props.reducerKeys).length > 0
-        && props.reducerKeys[key]) {
+    if (props && props.reducerKeys) {
+        if (typeof props.reducerKeys === 'string' &&
+            props.reducerKeys.length > 0) {
```
Comment (bencripps) [inline src/util/stateGetter.js]: if `reducerKeys` is of type `string` and it's defined, won't it have a `length` > 0? Seems like even if you pass the reducerKey of `grid`, this check would return false?

**Comment by bencripps** [comment]
New changes look good. I will test this out tonight. Thanks!

**Comment by bencripps** [comment]
Changes look good, tests look good, and I've verified it works locally. Would you mind squashing your two commits into one?

Great PR!

**Comment by drownbes** [comment]
@bencripps As I know github have added squash on merge for pull request, aren't they?
https://github.com/blog/2141-squash-your-commits

**Comment by bencripps** [comment]
That's an awesome feature I didn't know about! Squashed & merged!

**Pull request merged**